sdk/client/go/README.md

```diff
@@ -50,7 +52,7 @@ func main() {
 		log.Fatal(err)
 	}
 
-	variantEvaluationResponse, err := evaluationClient.Variant(context.Background(), &evaluation.EvaluationRequest{
+	variantResult, err := evaluationClient.Variant(context.Background(), &evaluation.EvaluationRequest{
 		NamespaceKey: "default",
```

Comment:
now that we are only accepted a single namespace in the constructors, I don't think we should also require namespaceKey here from the user, we should automatically set the `namespaceKey` to the  'current namespace'

Reply:
same for Python too

Reply:
@markphelps Ahh good thought. Makes sense, will change this.